pkg/authorization/authorizer/adapter/authorizer.go

```diff
@@ -0,0 +1,53 @@
+package adapter
+
+import (
+	"errors"
+
+	"github.com/golang/glog"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kauthorizer "k8s.io/kubernetes/pkg/auth/authorizer"
+	"k8s.io/kubernetes/pkg/auth/user"
+
+	oauthorizer "github.com/openshift/origin/pkg/authorization/authorizer"
+)
+
+type AdapterAuthorizer struct {
+	originAuthorizer oauthorizer.Authorizer
+}
+
+func NewAuthorizer(originAuthorizer oauthorizer.Authorizer) (kauthorizer.Authorizer, error) {
+	return &AdapterAuthorizer{originAuthorizer}, nil
+}
+
+func (z *AdapterAuthorizer) Authorize(kattrs kauthorizer.Attributes) error {
+	// Create context with namespace and user info
+	ctx := kapi.NewContext()
+	ctx = kapi.WithNamespace(ctx, kattrs.GetNamespace())
+	ctx = kapi.WithUser(ctx, &user.DefaultInfo{
+		Name:   kattrs.GetUserName(),
+		Groups: kattrs.GetGroups(),
+	})
+
+	// Build origin authorizer attributes
+	oattrs := &oauthorizer.DefaultAuthorizationAttributes{
+		Verb:     kattrs.GetVerb(),
+		Resource: kattrs.GetResource(),
+		// APIVersion        string
```

Comment:
You going to add these to kube or shall I?

Add a TODO and explain the limitations in a header comment.